_events/upcoming-virtual-events-planned.adoc

```diff
@@ -0,0 +1,15 @@
+---
+layout: page
+name: Upcoming Virtual Events Planned
+host_info:
+date: 2024-11-14
+image: /assets/images/CalConnectEventTBD_0.jpg
+---
+
+== Upcoming Virtual Events Planned
+
+image::/assets/images/CalConnectEventTBD_0.jpg[]
+
+CalConnect will host some one or two day virtual events on specific topics. Any suggestions on appropriate topics are welcome. We have currently in mind:
+
```

Comment:
Missing lots of content here:

https://www.calconnect.org/events/upcoming-virtual-events-planned
<img width="686" alt="Screenshot 2024-12-10 at 4 36 59 PM" src="https://github.com/user-attachments/assets/9f9ec926-ced6-4018-8a63-c19557c1765f">